es-vue-base/src/lib-components/EsReview.vue

```diff
@@ -49,25 +61,108 @@
+                <span
+                    class="name-holder d-inline-block text-truncate pr-1"
+                    data-testid="subtext-test">
+                    <span class="d-none d-lg-inline-block">{{ updatedComment && commentLimit ? "Updated by" : "Posted by" }}</span>
+                    {{ reviewerName }}
+                </span>
+                <span class="date-holder">on {{ updatedComment && commentLimit ? updatedDate : formattedDate }}</span>
+            </small>
+            <div
+                v-if="certified && !response && commentLimit"
+                class="d-lg-none d-flex flex-nowrap align-items-center ml-auto font-size-sm">
+                <div class="mr-2">
+                    Verified
+                </div>
+                <IconVerified
+                    class="text-gray-900"
+                    width="16px"
+                    height="16px" />
+            </div>
+        </div>
+        <div
+            v-if="certified"
+            class="d-none d-lg-flex text-gray-800 mt-2 align-items-center font-size-sm">
             <IconVerified
-                class="text-gray-900 mr-1"
+                class="text-gray-900 mr-2"
                 width="16px"
                 height="16px" />
             Verified Shopper
-        </small>
+        </div>
+        <div
+            v-if="response && commentLimit"
+            class="d-flex flex-nowrap align-items-center font-size-sm mt-2 mt-lg-3">
+            <b-link
+                data-testid="view-response"
+                @click="$emit('showMore')">
+                View provider response
+            </b-link>
+            <div
+                v-if="certified"
+                class="d-lg-none d-flex flex-nowrap align-items-center ml-auto text-gray-800">
+                <div class="mr-2">
+                    Verified
+                </div>
+                <IconVerified
+                    class="text-gray-900"
+                    width="16px"
+                    height="16px" />
+            </div>
+        </div>
+        <div
+            v-if="response && !commentLimit"
+            data-testid="developer-response">
+            <div
+                class="mt-3 d-flex flex-nowrap align-items-lg-start align-items-center">
+                <b-img
+                    class="logo p-0 mr-2 mr-lg-3"
+                    :src="response.developer_logo"
+                    :alt="`${developerName} logo`" />
+                <div>
+                    <p class="font-weight-bolder m-0 mb-lg-2">
+                        <span class="d-sm-inline-block">Response from </span>
+                        {{ developerName }}
+                    </p>
+                    <div
+                        class="d-none d-lg-block">
+                        {{ response.response_text }}
+                        <p class="font-size-sm mt-3">
+                            {{ `Responded on ${responseDate}` }}
```

Comment:
TIL you could use this syntax. Below is a bit more "normal".

```suggestion
                           Responded on {{ responseDate }}
```